packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -38,7 +38,7 @@ async function failIfSecretMissing(secretName: string, namespace: string) {
 async function copySecret(secretName: string, srcNamespace: string, destNamespace: string) {
   console.info(`Copying secret ${secretName} from namespace ${srcNamespace} to ${destNamespace}`)
   await execCmdWithExitOnFailure(`kubectl get secret ${secretName} --namespace ${srcNamespace} -o yaml |\
-  grep -v creationTimestamp | grep -v resourceVersion | grep -v selfLink | grep -v uid |\
+  grep -v creationTimestamp | grep -v resourceVersion | grep -v selfLink | grep -v uid | grep -v namespace |\
```

Comment:
Nit: This could be consolidated.
```suggestion
  grep -Ev 'creationTimestamp|resourceVersion|selfLink|uid|namespace' |\
```

Reply:
Ah cool, thanks